recipes/surfinBH/meta.yaml

```
@@ -0,0 +1,54 @@
+{% set name = "surfinBH" %}
+{% set version = "1.0.9" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 9d3df12a256e06c2c3d72983ac8355f02184d0faf2d8c8ac7728585b8b8effc5
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: true  # [win]
```

This looks like it can be noarch python? It's okay if the Windows CI then fails as the feedstock will only use linux:
```suggestion
  noarch: python
```

---

Thanks for the help! I didn't know the CI would do this.